front/pages/api/w/[wId]/assistants/[aId]/index.ts

```diff
@@ -0,0 +1,73 @@
+import * as t from "io-ts";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { getAgentConfiguration } from "@app/lib/api/assistant/configuration";
+import { Authenticator, getSession } from "@app/lib/auth";
+import { ReturnedAPIErrorType } from "@app/lib/error";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+
+export type GetAssistantResponseBody = {
+  assistant: AgentConfigurationType;
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<GetAssistantResponseBody | ReturnedAPIErrorType | void>
+): Promise<void> {
+  const session = await getSession(req, res);
+  const auth = await Authenticator.fromSession(
+    session,
+    req.query.wId as string
+  );
+  const owner = auth.workspace();
+  if (!owner) {
+    return apiError(req, res, {
+      status_code: 404,
+      api_error: {
+        type: "workspace_not_found",
+        message: "The workspace you're trying to modify was not found.",
+      },
+    });
+  }
+
+  if (!auth.user()) {
```

Comment:
Let's check the user is a builder (at least for now)?

Reply:
I allowed for users because it's a GET, so not really risky to open to users. I initially planned to allow GET for all and POST for builder only. But I added a commit to make the change!